.github/actions/setup-sui/action.yaml

```diff
@@ -1,15 +1,16 @@
 name: Setup Sui CLI
```

Comment:
this file will be deleted now?

Reply:
yeah, should be deleted after it works. i keep it as a reference while i'm working on it.